### PR TITLE
HBASE-25331 [HBOSS] TreeLockManager should log warnings about lock co…

### PR DESCRIPTION
**Pull request by wchevreuil**
…ntention periodically

Change-Id: I2e88d8647531928b30ba29e730b5a83b86cf4eb4

### CODE REVIEW

File: hbase-oss/src/main/java/org/apache/hadoop/hbase/oss/Constants.java [hunk 1]
```diff
@@ -32,4 +32,6 @@
   public static final String ZK_MAX_RETRIES = "fs.hboss.sync.zk.sleep.max.retries";
 
   public static final String CONTRACT_TEST_SCHEME = "fs.contract.test.fs.scheme";
+
+  public static final String WAIT_INTERVAL_WARN = "fs.hboss.lock-wait.interval.warning";
```
Comment (petersomogyi) [inline hbase-oss/src/main/java/org/apache/hadoop/hbase/oss/Constants.java:36]: This configuration name is defined but it is not read from the configuration. Initialize waitIntervalWarn from the config and fall back to the default value.

**Review by petersomogyi — CHANGES_REQUESTED** [review]

File: hbase-oss/src/main/java/org/apache/hadoop/hbase/oss/Constants.java (hunk 1 above)
Comment (wchevreuil) [reply hbase-oss/src/main/java/org/apache/hadoop/hbase/oss/Constants.java:36]: It is read on the impl class, _ZKTreeLockManager.initialize_ method, as _TreeLockManager_ is abstract and doesn't have an init/constructor method.

Comment (petersomogyi) [reply hbase-oss/src/main/java/org/apache/hadoop/hbase/oss/Constants.java:36]: I missed that part, already marked ZKTreeLockManager as Viewed when I searched for the config usage. 🙂

**Review by petersomogyi — APPROVED** [review]

**Comment by petersomogyi** [comment]
Please remove the Change-Id from commit message before merge.

**Review by joshelser — COMMENTED** [review]
Belated review. Thanks for making this quality of life change, Wellington.

My only worry is spinning-fast through this check. currentTimeMillis seems to be in the same order of magnitude as nanoTime, so we're doing things as efficiently as possible, I guess. Maybe eventually, we come back and do some kind of async model (so we don't busy-wait). Later :)

**Pull request merged**